public/app/percona/pmm-dump/PmmDump.types.ts

```diff
@@ -0,0 +1,66 @@
+export enum DumpStatus {
+  BACKUP_STATUS_INVALID = 'BACKUP_STATUS_INVALID',
```

Comment:
Is this correct for dumps that it uses backup statuses? (If it's the BE response like that, it's ok but I would still rename keys to match - BACKUP_STATUS_INVALID -> DUMP_STATUS_INVALID etc.)

Reply:
I didn't want to have a code duplication. But you're right , we should have separate statuses for dumps.

Reply:
The response from the backend contains "BACKUP_" , but I've renamed the enums